server/controllers/todoControllers.js

```diff
@@ -0,0 +1,68 @@
+import asyncHandler from "express-async-handler"
+import { Todo } from "../models/todoModel.js"
+
+// @desc    Get all todos
+// @route   GET "/api/todo/"
+const getAllTodos = asyncHandler(async (req, res) => {
+    const getTodos = await Todo.find().sort({ createdAt : -1 })
+
+    if(getTodos){
+        res.status(201).json({message: "All todos fetched", data: getTodos})
+    }else{
+        res.status(500).json({error : "Server error"})
+    }
+})
+
+// @desc    Create a new todo
+// @route   POST "/api/todo/createTodo"
```

Comment:
the route should be api/todo, in REST API post request means its a create fuctionality